dcs/src/main/java/org/trafodion/dcs/server/ServerManager.java

```diff
@@ -400,7 +415,9 @@ public Boolean call() throws Exception {
             // /bin/dcs-daemon script
             // which DOES NOT set childServers count.
             for (int childInstance = 1; childInstance <= childServers; childInstance++) {
-                serverHandlers[childInstance-1] = new ServerHandler(childInstance);
+                ServerHandler serverHandler = new ServerHandler(childInstance);
+                serverHandler.setRestartAttempts(this.restartAttempts);
```

Comment:
suggest to add a new constructor new ServerHandler(this.conf, childInstance), then if you add more KVs in the future, the only changes may from ServerHandler